### PR TITLE
install django rest framework 

### PR DESCRIPTION
**Pull request by mengxiaoguai**
1. Installed djangorestframework. 
2. Created accounts app

### CODE REVIEW

File: accounts/api/serializers.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.contrib.auth.models import User, Group
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = User
+        fields = ['url', 'username', 'email']
```
Comment (92fueler) [inline accounts/api/serializers.py:8]: you'd be better to have newline at the end of the file

File: twitter/urls.py [hunk 2]
```diff
@@ -14,8 +14,15 @@
...
+
+router = routers.DefaultRouter()
+router.register(r'api/users', views.UserViewSet)
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-]
+    path('', include(router.urls)),
+    path('api-auth/', include('rest_framework.urls', namespace='rest_framework')),
+]
```
Comment (92fueler) [inline twitter/urls.py:28]: you'd be better to add newline at the end of the file

**Review by 92fueler — CHANGES_REQUESTED** [review]

File: accounts/api/serializers.py (hunk 1 above)
Comment (mengxiaoguai) [reply accounts/api/serializers.py:8]: ok, will do.

File: twitter/urls.py (hunk 2 above)
Comment (mengxiaoguai) [reply twitter/urls.py:28]: sure, will do.

**Review by 92fueler — APPROVED** [review]
lgtm

**Pull request merged**